testutil/docker.go

```diff
@@ -271,3 +271,24 @@ func DockerInspect(containerID string) (types.ContainerJSON, error) {
 	x.Check(err)
 	return cli.ContainerInspect(context.Background(), containerID)
 }
+
+// checkHealthContainer check health of container and determine wheather container is ready to accept request or not
+func CheckHealthContainer(socketAddrHttp string) error {
+	var err error
+	var resp *http.Response
+	for i := 0; i < 30; i++ {
+		resp, err = http.Get("http://" + socketAddrHttp + "/health")
```

Comment:
you can setup the URL once, don't need to do it in each iteratino

Reply:
done